app/controllers/users_controller.rb

```diff
@@ -30,6 +30,10 @@ def show
 
   # GET /users/1/edit
   def edit
+    if @user.password_digest.blank?
```

Comment:
this is kind of a hack. We might need a way to differentiate local and net id accounts in the future @boyangles